ocaml/idl/datamodel.ml

```diff
@@ -433,7 +433,10 @@ module Pool_update = struct
       ~doc:"Attach the pool update VDI"
       ~in_oss_since:None
       ~in_product_since:rel_ely
-      ~params:[ Ref _pool_update, "self", "The update to be attached"]
+      ~versioned_params:[
+        {param_type=Ref _pool_update; param_name="self"; param_doc="The update to be attached"; param_release=ely_release; param_default=None};
+        {param_type=Bool; param_name="use_localhost_proxy"; param_doc="Use the localhost proxy"; param_release=ely_release; param_default=Some (VBool false)};
```

Comment:
Is `ely_release` correct here, or should we specify naples_release here and havana_release in the backport?

Reply:
You are right, but I don't find anywhere define `havana_release` in 1.14-lcm which is havana branch. Do we change the mechanism for the release definition?

Reply:
Please resolve this when you changing it to `naples_release`